### PR TITLE
[Spec] Add flatbuf devel dependency for tflite

### PR DESCRIPTION
**Pull request by gichan-jang**
Add flatbuf devel dependency for tflite

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [* ]Passed [ ]Failed []Skipped
2. Run test: [* ]Passed [ ]Failed []Skipped

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -42,6 +42,11 @@
 %define		vivante_support 0
 %endif
 
+# Flatbuffers-dev package is necessary to build tensorflow-lite
+%if 0%{?tensorflow_lite_support}
+%define		flatbuf_support 1
```
Comment (dongju-chae) [inline packaging/nnstreamer.spec:47]: Line 9 also defines `flatbuf_support 1`. How about using if/else statements like other `*_support` variables.

Comment (myungjoo) [reply packaging/nnstreamer.spec:47]: NO. tensorflow-dev should get flatbuf-dev automatically. nnstreamer.spec should never take care of such secondary dependencies.

Comment (dongju-chae) [reply packaging/nnstreamer.spec:47]: Yes, makes sense.

Comment (gichan-jang) [reply packaging/nnstreamer.spec:47]: Thank you.
I addded `flatbuffers-devel` dependency to `tensorflow-lite-devel`, I close this PR.
Please review this : https://review.tizen.org/gerrit/#/c/platform/upstream/tensorflow/+/236267/

**Pull request closed without merging**